tests/run-android-64.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
Perhaps naming this `run-android-arm64` would be better? We can rename the other one also while at it.